imports/email-templates/case-new-message.js

```diff
@@ -1,5 +1,18 @@
 import url from 'url'
 import { optOutHtml, optOutText } from './components/helpers'
+import { matchWidth } from '../util/cloudinary-transformations'
+
+function linkAttachment (message) {
+  const msgParts = message.split(' ')
+  if (msgParts.length > 1 && msgParts[0] === '[!attachment]') {
```

Comment:
You should use https://github.com/unee-t/frontend/blob/master/imports/util/matchers.js#L3 instead of writing this check on you own

Reply:
I didn't actually understand your code. attachmentTextMatcher is a test to see if it's off that type? Never seen a base64 type.

And then there is still the bit where the second part if properly checked since it can be truncated and put into a template, unlike here:

https://github.com/kaihendry/frontend/blob/f66e4f42373df558aba48ae577d178d45ec470b8/imports/ui/case/case-details.jsx#L384

Reply:
The base64 type is for messages while they're being uploaded... 
I brought this up because you just shouldn't reinvent the matching and parsing of the attachment url IMO. 
The second part? You mean checking for a file extension that matches an image?
We should have only image type attachments for now, but if you think this improves the robustness of the check, I think you should still use `attachmentTextMatcher` but modify it so other places in the code can benefit from it as well :smile:

Reply:
Ok, will re-try tomorrow.